src/sql/ResourceSelector.test.tsx

```diff
@@ -24,10 +25,10 @@ describe('ResourceSelector', () => {
     render(<ResourceSelector {...props} default="foo" value={defaultKey} fetch={fetch} onChange={onChange} />);
     expect(screen.queryByText('default (foo)')).toBeInTheDocument();
 
-    if (!props.label) {
-      throw new Error('label is required');
+    if (!props['aria-label']) {
+      throw new Error('aria label is required');
     }
-    const selectEl = screen.getByLabelText(props.label);
+    const selectEl = screen.getByLabelText(props['aria-label']);
```

Comment:
Imo it's a bit easier to understand what's happening in a test here to say:
```const selectEl = screen.getByLabelText("the actual label");```

but a bit nitpicky of me!